libdnf/repo/solv_repo.cpp

```diff
@@ -421,7 +421,8 @@ void SolvRepo::load_system_repo(const std::string & rootdir) {
 
     int solvables_start = pool->nsolvables;
 
-    int flagsrpm = REPO_REUSE_REPODATA | RPM_ADD_WITH_HDRID | REPO_USE_ROOTDIR;
+    // TODO(egoode) investigate performance hit of RPM_ADD_WITH_CHANGELOG, possibly make this configurable
+    int flagsrpm = REPO_REUSE_REPODATA | RPM_ADD_WITH_HDRID | REPO_USE_ROOTDIR | RPM_ADD_WITH_CHANGELOG;
```

Comment:
This change causes changelog data to always be loaded for the system repository. I'm not sure what the performance implications are; there is no measurable difference on my system, but I don't have a lot of packages installed. We may want to go ahead and make this function more configurable, or we can go ahead with this change and possibly revisit in the future.

Reply:
I guess the TODO is OK there and when we'll encounter any related issue in the future, it could be revisited then as you've written. Another related issue is that we load all metadata for the system repository when calling `Repo::load_system_repo()`, and there is already a TODO present regarding this.